src/wallet.ts

```diff
@@ -134,18 +134,35 @@ class WalletLedger {
     };
   }
 
-  async importFromSerializedAccount(account: SerializedAccount): Promise<Account> {
+  // eslint-disable-next-line class-methods-use-this
+  exportToSerializedAccountSync(account: Account): SerializedAccount {
+    const data = account.xpub.storage.exportSync();
```

Comment:
TS complains about possibly undefined methods, here and for other import/export calls.